desc/plotting.py

```diff
@@ -599,6 +599,10 @@ def plot_2d(
         * ``ylabel_fontsize``: float, fontsize of the ylabel
         * ``cmap``: str, matplotlib colormap scheme to use, passed to ax.contourf
         * ``levels``: int or array-like, passed to contourf
+        * ``field``: MagneticField, a magnetic field with which to calculate Bn on
```

Comment:
Idea: to better match the Objectives API, we could change the first argument `eq` to `things`, which would normally be the equilibrium but in this case would be the tuple `(eq, field)`. Instead of adding a new kwarg. And this would also be more general for adding new plot quantities in the future.

Reply:
This is a good idea, but not necessarily needed by this PR. I will make an issue for it